src/utils/hooks/useLocalStorage.ts

```diff
@@ -0,0 +1,30 @@
+export default function useLocalStorage() {
+	// biome-ignore lint/suspicious/noExplicitAny: can pass anything
```

Comment:
todo: You can avoid this TS error by usingi `unknown`

Reply:
I didn't know about unknown. I need to look at that. 

I'm probably going to delete this file. I don't need a hook as Local Storage is accessed only one place, one time.

Reply:
Fix it for now, and then decide later if you'll delete it or not :)

Reply:
DONE!!